tests/duty.data.ts

```diff
@@ -0,0 +1,25 @@
+import { ObjectId } from 'mongodb';
+import { DutySchema } from '../services/schemas/duty.zschema';
+
+const createTestDuties = (amount: number) => {
```

Comment:
`CreateRandomDuty` and create one each test - will allow you to test in parallel

Reply:
resolved